### DISCUSSION TITLE
Intruducing new Botan binding to Janet.

### DISCUSSION THREAD

**Discussion by tttuuu888** [randombit/botan#3982] [General]
First of all, thank you for @randombit and all Botan contributors.
Botan has been my favorite cryptography library for many years.
Recently I found new interesting language, Janet. And the first thing came to mind was, making bindings to Botan.
Now that I feel like most of FFI bindings are complete, I wanted to share my work and say thank you to the Botan community.

You can find some more information from the below link.

https://tttuuu888.github.io/janet-botan/

I would appreciate for any feedback.

Thank you.

**Comment by reneme** [comment]
Interesting! Thanks for sharing your work. Your wrapper seems to be pretty comprehensive. That's impressive!
Frankly, I've never heard of Janet, but I'd be interested in the sort of use cases you're using it for.

**Reply by tttuuu888** [reply]
Hi @reneme, thank you for your interest in my work.
Janet has the advantage of being a lisp-like language while also being easy to build executable on.
I like the Lisp languages, and I was attracted to Janet's syntactic features and the ease of creating executable files.
Janet is a new language for me as well, so there are not many uses cases yet.
I have been using Botan to pre-implement or verify encryption-related functions of embedded devices, and now I think I will start to use Janet-botan for this purpose.
